src/ex1.py

```diff
@@ -0,0 +1,22 @@
+
+
+week = input("Enter week number(1-7): ")
+
+match week:
+    case "1":
+        print("Monday")
+    case "2":
+        print("Tuesday")
+    case "3":
+        print("Wednesday")
+    case "4":
+        print("Thursday")
+    case "5":
+        print("Friday")
+    case "6":
+        print("Saturday")
+    case "7":
+        print("Sunday")
+    case _:
+        print("Invalid input! Please enter week number between 1-7")
+    
```

Comment:
not bad 👍 

prova a mettere tutto dentro una funzione e usa https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do